src/styles/blocks/anti-snake.scss

```diff
@@ -0,0 +1,41 @@
+$colors:
+  1 #f00,
+  2 #c00,
+  3 #900,
+  4 #600,
+  5 #300,
+  6 #000;
+
+.anti-snake {
+  &__container {
+    display: grid;
+
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+
+    @media (max-width: 599px) {
+      grid-template-columns: 1fr;
+      grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
+    }
+
+    @media (max-width: 899px) and (min-width: 600px) {
```

Comment:
If in future size of blocks will change, these queries will break. Do it more adaptive next time